compositor_render/src/wgpu/ctx.rs

```diff
@@ -69,6 +69,23 @@ impl WgpuCtx {
             error!("Selected adapter is CPU based. Aborting.");
             return Err(CreateWgpuCtxError::NoAdapter);
         }
+        let required_features =
```

Comment:
I did not comment specifically on each variable, but I would expect that if you change this variable name that you also will change `missing_critical_features`.
The same applies to other cases like this.